### DISCUSSION TITLE
Do KEDA scalers have published schemas for what sort of metadata is required and/or optional?

### DISCUSSION THREAD

**Discussion by sabbour** [kedacore/keda#3882] [Q&A / Need Help]
Do KEDA scalers have published schemas for what sort of metadata is required and/or optional?

**Comment by tomkerkhove** [comment]
Short answer, no but we are considering it: https://github.com/kedacore/keda/issues/2359

Our documentation provides an overview though. May I ask why you are asking?

**Reply by sabbour** [reply]
Sorry for not responding earlier. Main reason was to create tooling to write up the KEDA YAML files considering user input.

**Reply by tomkerkhove** [reply]
Agreed, that is the goal of the issue above indeed so I suggest you join the fun there!
